api/src/main/kotlin/eu/codeloop/thehub/properties/JobProperties.kt

```diff
@@ -0,0 +1,8 @@
+package eu.codeloop.thehub.properties
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "thehub.job")
+class JobProperties {
+    lateinit var `approved-at-default`: String
```

Comment:
Note: You could use `approvedAtDefault` - Spring has ability to parse properties that are "dahsed" or "camelCase", so

```
job.approved-at-default: 2000-01-01T00:00:00.000Z
```

is equal to

```
job.approvedAtDefault: 2000-01-01T00:00:00.000Z
```

Reply:
OK.